blazor/ClientSideExample/ClientSideExample/ViewModels/FetchDataViewModel.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System.Collections.Generic;
+using System.Net.Http;
+using System.Reactive;
+using System.Threading.Tasks;
+using ClientSideExample.Data;
+using Microsoft.AspNetCore.Components;
+using ReactiveUI;
+
+
+namespace ClientSideExample.ViewModels
+{
+    public class FetchDataViewModel : ReactiveObject
+    {
+        private List<WeatherForecast> _forecasts = new List<WeatherForecast>();
+        private readonly HttpClient _http;
+        public FetchDataViewModel(HttpClient http)
+        {
+            _http = http;
+            LoadForecasts = ReactiveCommand.CreateFromTask(LoadWeatherForecastsAsync);
```

Comment:
Pipe `LoadForecasts` into a `ToProperty` and turn Forecasts into an OAPH

Reply:
Done